### DISCUSSION TITLE
Problems debugging stowed exceptions

### DISCUSSION THREAD

**Discussion by e0ff** [microsoft/microsoft-ui-xaml#9042] [Q&A]
We are running into problems with our WinUI 3 app. The users are randomly getting crashes most of these crashes are the result of stowed exceptions. I have crash dumps from when these crashes have happened but can't get the stowed exception information from them. I've installed the PDE plugin but I can't get it to work. Here is an example of what I'm doing/what I'm getting:

```
This dump file has an exception of interest stored in it.
The stored exception information can be accessed via .ecxr.
(6cc.2a1c): Unknown exception - code c000027b (first/second chance not available)
For analysis of this file, run !analyze -v
KERNELBASE!RaiseFailFastException+0x152:
00007ffc`afe6c242 0f1f440000      nop     dword ptr [rax+rax]
0:000> .exr -1
ExceptionAddress: 00007ffc605095e4 (Microsoft_ui_xaml!FailFastWithStowedExceptions+0x0000000000000058)
   ExceptionCode: c000027b
  ExceptionFlags: 00000001
NumberParameters: 2
   Parameter[0]: 00000288adeff940
   Parameter[1]: 0000000000000002
0:000> !pde.dse
=========================================================================================
 PDE v11.3 - Copyright 2017 Andrew Richards
=========================================================================================
Usage: !dse [addr]
0:000> !pde.dse 00007ffc605095e4
```

I've tried other addresses but I still get no output and I've tried adding 0x in front of the address but it also still doesn't work. Anyone know of another way to get the stowed exception information or how to get the PDE plugin to work?

**Comment by pszyjaciel** [comment]
try this instead: !pde.dse 00000288adeff940
The address is the Parameter[0]. The command dse will show you the call stack for this crash state. In my case I could figure out, that a xaml control CommandBarFlyoutCommandBar is causing the stowed exception. Removed the control and exception was gone.

**Reply by e0ff** [reply]
I think I tried that before and it didn't work, I eventually did get some useful information from them but had to turn on full memory dumps. After turning that on the !pde.dse command worked.
